src/dialog/an-dialog.factory.js

```diff
@@ -151,6 +160,53 @@
                         });
                     };
 
+                    var getFocusableElements = function() {
```

Comment:
If a function starts with _get_, you usually don't expect it to have side-effects and expect it to return something. I suggest that you move the code from this function into the _setInitialFocusableElements_ function.

Reply:
hm, yeah it seems the name of this function can be tricky since it gets all focusable elements and in the same set it to the internal variable of the class. Ok I will move it